### PR TITLE
GA migration PostCommit SQL and MongoDBIT

### PR DESCRIPTION
**Pull request by benWize**
As part of the migration of Precommit and Postcommit Jobs from Jenkins to GA in self-hosted runners, this PR contains:

- Migrated workflow [job-postcommit-python-mongodbit.yml](https://github.com/benWize/beam/blob/182a5e454e2d8072ef03b0dac45859b04a072557/.github/workflows/job-postcommit-python-mongodbit.yml)
- Migrated workflow [job-postcommit-sql.yml](https://github.com/benWize/beam/blob/182a5e454e2d8072ef03b0dac45859b04a072557/.github/workflows/job-postcommit-sql.yml)

The migrated workflows were removed from [.test-infra/jenkins/README.md](https://github.com/benWize/beam/blob/182a5e454e2d8072ef03b0dac45859b04a072557/.test-infra/jenkins/README.md) and added to [CI.md](https://github.com/benWize/beam/blob/182a5e454e2d8072ef03b0dac45859b04a072557/CI.md)

**DO NOT MERGE** until the effort to use **self-hosted runners** is completed https://github.com/apache/beam/pull/22703

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by benWize** [comment]
Successful executions on self-hosted runners:

- SQL: https://github.com/benWize/beam/actions/runs/3040953057
- MongoDBIOIT: https://github.com/benWize/beam/actions/runs/3040953056

**Comment by fernando-wizeline** [comment]
LGTM!

File: .github/workflows/job-postcommit-python-mongodbit.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# under the License.
+
+# This workflow will run the integration test of python mongodbio class.
+
+name: Python MongoDBIO IT
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
```
Comment (damccorm) [inline .github/workflows/job-postcommit-python-mongodbit.yml:25]: Is there a reason we added the cron job to these 2 jobs? We don't run these on a schedule currently, right?

Comment (benWize) [reply .github/workflows/job-postcommit-python-mongodbit.yml:25]: These jobs run on a scheduled basis in the current configuration

https://ci-beam.apache.org/job/beam_PostCommit_Python_MongoDBIO_IT/
https://ci-beam.apache.org/job/beam_PostCommit_SQL/

Comment (damccorm) [reply .github/workflows/job-postcommit-python-mongodbit.yml:25]: Oh, you're right I misread this. It looks like they don't run on push though, or am I missing something else? If so, should we remove the `push` trigger?

Comment (benWize) [reply .github/workflows/job-postcommit-python-mongodbit.yml:25]: Yes, we can remove the push trigger

**Pull request closed without merging**